src/Resources/Locales/en.json

```diff
@@ -952,5 +952,10 @@
             "3dAssetLoadingGenericErrorMessage": "Error loading model. Try clearing your cache and try again or check your access for that file.",
             "3dAssetDimensionError": "There may be an issue with your model dimensions, please check there are no erroneous meshes."
         }
+    },
+    "advancedSearch": {
+        "toggleSelection": "Toggle selection",
```

Comment:
can we name the keys for what they are used for rather than the string they represent? ex: "rowHoverText"